### ISSUE TITLE
Question: reduce GeoJson sizes / cleanup coast 

### ISSUE THREAD

**Issue by HugoGresse** [tdwg/wgsrpd#9]
Hi! 

We are working to integrate the plants distributions from WGSRPD in Pl@ntNet services. (level2)

We are finalizing the work on our end but still have some issues:

1. The coast definition are not exact at all. For example, in Marseille, France, part of the city is outside France: 
![Capture d’écran 2022-06-15 à 14 50 29](https://user-images.githubusercontent.com/662377/173831436-8d048b5b-1ef0-47ea-9d8c-29c0f66d0c6b.png)
And for Corsica, France: 
![Capture d’écran 2022-06-15 à 14 53 40](https://user-images.githubusercontent.com/662377/173831733-3f232196-620c-4eee-a7de-ca2c977f9845.png)

2. The amount of details on other coasts are at the same time big and not very accurate: 
![Capture d’écran 2022-06-15 à 14 50 44](https://user-images.githubusercontent.com/662377/173831636-b04a6fd4-83b3-40d5-924a-589221e60a4e.png)
This results in very low performance when trying to display the GeoJson en embedded map on mobile devices. 

My question is, would you be open to accept PR which improve this? 

- reduce the amount of point on coasts 
- keep every country land within the boundary of the defined area

**Comment by peterdesmet** [comment]
Hi @HugoGresse, I'm not maintaining the standard, but that seems like a good idea. Thanks for your contribution!

**Comment by HugoGresse** [comment]
Thanks you @peterdesmet for the quick answer. 

Would updating only the .geojson and not the other formats be ok on your side? I mean would you be able to merge a PR doing only that?

**Comment by peterdesmet** [comment]
Yeah, I think reducing the size of geojson only would be fine. Also notifying maintainers @tdwg/wgsrpd @PacoPando and @NicBailly.
